src/server/src/main/java/io/cassandrareaper/core/ClusterProperties.java

```diff
@@ -23,22 +23,36 @@
 @JsonDeserialize(builder = ClusterProperties.Builder.class)
 public final class ClusterProperties {
   private final int jmxPort;
+  private final String jmxUsername;
+  private final String jmxPassword;
```

Comment:
why not just store the `jmxCredentials` here instead? (then we wouldn't have to build it everytime in the cluster's getter method)

Reply:
ahh, good idea.  absolutely.